myia/ir/print.py

```diff
@@ -131,7 +149,7 @@ def __init__(self):
         )
 
     def __call__(self, node):
-        if node.is_constant_graph():
+        if isinstance(node, Node) and node.is_constant_graph():
```

Comment:
This should always be called with a Node so that check should not necessary

Reply:
It is now used for graph_printer as well, which passes Graphs sometimes.

Reply:
Ok then.